packages/firestore/src/remote/persistent_stream.ts

```diff
@@ -318,8 +318,9 @@ export abstract class PersistentStream<
       // If this is an intentional close ensure we don't delay our next connection attempt.
       this.backoff.reset();
     } else if (error && error.code === Code.RESOURCE_EXHAUSTED) {
-      log.debug(
-        LOG_TAG,
+      // Log the error. (Probably either 'quota exceeded' or 'max queue length reached'.)
```

Comment:
Oh. Good call logging the actual error (and commenting the justification). Thanks!  Were you able to verify that this logs sanely? I think our logger just outputs the JSON representation for objects, which may be wonky. It may be better to do error.toString() ?

Reply:
Yeah, it looks reasonable to me:

```
Firestore (4.8.0) 2017-12-13T16:41:36.587Z: FirebaseError: [code=resource-exhausted]: Stream reached its maximum queuing limit, and aborted.
Firestore (4.8.0) 2017-12-13T16:41:36.590Z: Using maximum backoff delay to prevent overloading the backend.
```

Reply:
👍

Reply:
Turns out that .toString() is needed anyways due to the TS tests. Logs still look the same.